### ISSUE TITLE
int8conv_cuda.group_conv arguments mismatch

### ISSUE THREAD

**Issue by tshadley** [wangmaolin/niti#6]
Quick note that these calls are mismatched, stride and dilation swapped.

https://github.com/wangmaolin/niti/blob/d09733516edb524531d6fa50b96612bf13c731e2/ti_torch.py#L214-L220

https://github.com/wangmaolin/niti/blob/d09733516edb524531d6fa50b96612bf13c731e2/pytorch/int8conv-extension/int8conv.cpp#L11-L17

**Comment by lgrativol** [comment]
@tshadley Hello, have you ever managed to reproduce the results of the paper ? I was trying to do and with the base script, i get 10 -12 % in accuracy.

**Comment by tshadley** [comment]
> @tshadley Hello, have you ever managed to reproduce the results of the paper ? I was trying to do and with the base script, i get 10 -12 % in accuracy.

Yes,  on compute 7.5 (Turing).   Got 90.46% around epoch 166:
```
2023-04-11 16:18:49 - INFO - best_prec1: 90.460000 ./results/test
2023-04-11 16:18:49 - INFO - Epoch: 166 Train Prec@1 99.066 Train Loss 0.027 Valid Prec@1 90.460 Valid Loss 0.521 
```

**Comment by lgrativol** [comment]
Ahh ok, thanks for the reply !! I'm trying  on GPU Ampere, using the last version of the repo. Well, I'm gonna keep trying to debug it. :)
